src/open-websocket.js

```diff
@@ -0,0 +1,14 @@
+import Html5WebSocket from 'html5-websocket'
+import ReconnectingWebSocket from 'reconnecting-websocket'
+import WebSocket from 'ws'
+
+export default wsAddressBuilder => {
+  return new ReconnectingWebSocket(wsAddressBuilder, undefined, {
```

Comment:
This second parameter is the optional array of protocols right? Can we pass `null` instead?

Reply:
Yes, it defines the protocols. I used `undefined` because the examples of the project use `undefined` too: https://github.com/pladaria/reconnecting-websocket#using-alternative-constructor 

Why do you want it to be `null`?

Reply:
I was looking at another project following the same name. Generally I prefer `null` over `undefined` since we know this parameter exists but has no value. The `window` check with `undefined` is okay since we don't really know if it exists or not. This is more a convention than anything